src/com/estafet/learning/US008/Invoice.java

```diff
@@ -0,0 +1,27 @@
+package com.estafet.learning.US008;
+
+import java.util.Objects;
+
+public abstract class Invoice extends Order{
+    public static final String INVOICE_NAME = "INV";
+    public String OrderNumber;
+    public String ClientDetails;
+    public String ListWithArticles;
+    public String TotalAmount;
+}
+class TradeInvoice extends Order {
```

Comment:
it's incorrect, should extend Invoice class

Reply:
changed.